MerlinAU.sh

```diff
@@ -8722,7 +8722,7 @@ _ShowAdvancedOptionsMenu_()
        fi
        if "$sendEMailNotificationsFlag"
        then
-           printf "\n${padStr}[Currently ${GRNct}ENABLED${NOct}, Format: ${GRNct}${sendEMailFormaType}${NOct}]\n"
```

Comment:
The closing right square bracket char is missing!
```
printf "\n${padStr}[Currently ${GRNct}ENABLED${NOct}]\n"
                                                   ^^^
```

Reply:
This is why you should never do an in-place edit on your phone! :P The other PR has it properly

Reply:
@Martinski4GitHub 

Fixed :) (using inplace edit on my phone)
Do what I say, not what I do? Is that how it goes? LOL